core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -862,19 +839,29 @@ extern "C" fn new_confirm_modify_fee(n_args: usize, args: *const Obj, kwargs: *m
         let sign: i32 = kwargs.get(Qstr::MP_QSTR_sign)?.try_into()?;
         let user_fee_change: StrBuffer = kwargs.get(Qstr::MP_QSTR_user_fee_change)?.try_into()?;
         let total_fee_new: StrBuffer = kwargs.get(Qstr::MP_QSTR_total_fee_new)?.try_into()?;
+        let fee_rate_amount: Option<StrBuffer> = kwargs
+            .get(Qstr::MP_QSTR_fee_rate_amount)?
+            .try_into_option()?;
 
         let (description, change) = match sign {
             s if s < 0 => ("Decrease your fee by:", user_fee_change),
             s if s > 0 => ("Increase your fee by:", user_fee_change),
             _ => ("Your fee did not change.", StrBuffer::empty()),
         };
 
-        let paragraphs = Paragraphs::new([
-            Paragraph::new(&theme::TEXT_NORMAL, description.into()),
-            Paragraph::new(&theme::TEXT_MONO, change),
-            Paragraph::new(&theme::TEXT_NORMAL, "\nTransaction fee:".into()),
-            Paragraph::new(&theme::TEXT_MONO, total_fee_new),
-        ]);
+        let mut paragraphs_vec = ParagraphVecShort::new();
+        paragraphs_vec
+            .add(Paragraph::new(&theme::TEXT_BOLD, description.into()))
```

Comment:
Here we change `TEXT_NORMAL` to `TEXT_BOLD` - is it wanted?

Reply:
Figma has `TEXT_NORMAL`.